cli/listings.sh

```diff
@@ -218,13 +218,20 @@ pom_bw() {
     tmpfile=`gettmpfile`
     pushd "${pom_path}" > /dev/null
     mvn -q dependency:list -DoutputFile=$tmpfile -DappendOutput=true $mvn_opts
-    popd > /dev/null
 
     if [ $? -ne 0 ]; then
         rm $tmpfile
+        echo ""
+        echo ""
+        echo "================================================================="
+        echo "'mvn dependency:list' command failed."
+        echo "Consider running 'mvn clean install' before running the pom-bw command again to fix the issue"
+        echo "================================================================="
         exit
     fi
 
+    popd > /dev/null
```

Comment:
`popd` has to be moved because it interferes with the exit status check on line 222.